sources/web/datalab/polymer/test/file-browser-test.ts

```diff
@@ -100,4 +114,57 @@ describe('<file-browser>', () => {
       assert(!row.selected, 'file ' + i + ' should not be selected');
     });
   });
+
+  it('file operations are disabled when no items selected', () => {
+    const files: ItemListElement = testFixture.$.files;
+    files._unselectAll();
+    fileAddButtonIds.forEach((id) => {
+      const button = testFixture.$[id] as HTMLElement;
+      assert(!button.hasAttribute('disabled'), id + ' button should always be enabled');
```

Comment:
The button id names already include "Button", so you don't need that word in this message. Likewise in a few other places in this file.

Reply:
Done.